pyproject.toml

```diff
@@ -1,9 +1,10 @@
 [build-system]
 requires = [
-    "setuptools>=42",
+    "setuptools >= 42.0, < 60.0",
     "wheel",
     "cmake",
     "numpy",
+    "pyhepmc>=2.7.3"
```

Comment:
pyhepmc is not a build requirement. It is also not an install requirement, at least that's not the intention. pyhepmc should be optional, you should be able to use impy without it, but if it is installed, you can profit from extra features. pyhepmc should be only installed as a test requirement, specified in setup.cfg.

Reply:
Build requirements are stuff you need during the build. Install requirements are stuff you need so that `import impy` works and all imports of its submodules. Stuff that's only needed for specific functions to work are optional dependencies, which should be put into the test requirements.

Reply:
Also, why are you capping setuptools? I know NumPy recommends this, but that's only due to the fact they aren't interested in fixing numpy.distutils anymore and are removing it in favor of dropping setuptools entirely in favor of meson-python. If you do need the built-in distutils over the pypa one, there's an environment variable you can set in setup.py that does the same thing without the cap.